packages/documents/graphql/schema-types.js

```diff
@@ -38,19 +38,20 @@ type Meta {
   twitterDescription: String
   prepublication: Boolean
   publishDate: DateTime
-  template: String
   feed: Boolean
   kind: String
   color: String
   series: Series
   format: Document
-  # the discussion wrapping document
   dossier: Document
-  discussion: Document
-  # the id of the discussion itself
-  discussionId: ID
+
   credits: JSON
   audioSource: AudioSource
+
+  # template of the article
+  # if (and only if) this is 'discussion' the page must show
+  # the discussion component (in the bottom)
```

Comment:
I wouldn’t document frontend behaviours in the API. The frontend is free to do as it pleases 😘